pyproject.toml

```diff
@@ -27,14 +27,17 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 dependencies = [
-    "django>=4.2"
+    "django>=4.2",
+    "django-import-export",
+    "psycopg2",
```

Comment:
this breaks our ability to use `psycopg` instead of `psycopg2` - better to just document that PostgreSQL is required and not specify this dependency.

Reply:
right but this won't build now. It needs it here: https://github.com/maykinmedia/maykin-auth-matrix/actions/runs/10145474467/job/28051237009?pr=1 so if we can't do that I did not find a working alternative.

Reply:
are you suggesting we should remove all psycopg deps? 

How is the CI able to pass without having psycopg to dialog with the DB ?

Reply:
ensure that it's installed via `tox.ini`, like here: https://github.com/maykinmedia/django-digid-eherkenning/blob/master/tox.ini#L63